relayer/application_relayer.go

```diff
@@ -471,7 +472,14 @@ func (r *ApplicationRelayer) createSignedMessageAppRequest(
 		}
 		responseChan := r.network.Handler.RegisterRequestID(requestID, vdrSet.Len())
 
-		sentTo := r.network.Network.Send(outMsg, vdrSet, r.sourceBlockchain.GetSubnetID(), subnets.NoOpAllower)
+		sentTo := r.network.Network.Send(
+			outMsg,
+			// TODO: consider whether we need to specify the other
```

Comment:
My understanding is that the other fields are used to specify the number of additional nodes to sample. Since we are making requests to specific nodes explicitly, I don't think additional sampling is necessary.

Reply:
thanks for the input, removed the TODO in https://github.com/ava-labs/awm-relayer/pull/373/commits/e0794af65bdfa5c322347a8370272690cd00dbc4